### ISSUE TITLE
Categorical Variable That Has Different Levels in Train and Test Set

### ISSUE THREAD

**Issue by kaiweiang** [automl/auto-sklearn#496]
Hi again,

I'm aware categorical attributes will be automatically One-Hot encoded. When a dataset is split into train and test set, if somehow one categorical variable from the train set has fewer levels than from the test set, how is this issue being handled? The similar question can be found from this [stackoverflow](https://stackoverflow.com/questions/39804733/dummy-creation-in-pipeline-with-different-levels-in-train-and-test-set).

Would appreciate any explanation.

**Comment by mfeurer** [comment]
New levels will be ignored at test time because we do not have any information about them and can therefore not learn anything about them.

**Comment by kaiweiang** [comment]
> New levels will be ignored at test time 

Ignored as in predict the new levels data as negative class (in binary classification scenario)?
